pypit/tests/test_arparse.py

```diff
@@ -38,3 +38,14 @@ def test_load_spectrograph():
     assert spect._spect['keyword']['dispname'] == '01.GRISM_N'
     assert spect._spect['keyword']['dichroic'] == '01.BSPLIT_N'
     spect.save()
+
+def test_parse_binning():
+    """ Test parse binning algorithm
+    """
+    bin1, bin2 = arparse.parse_binning('2,2')
+    assert bin1 == 2
+    assert bin2 == 2
+    # Other input
+    bin1, bin2 = arparse.parse_binning((2,2))
```

Comment:
This is confusing.  I currently require the input to
be strings.  As these are not, it returns the default of 1,1.

have added a note in the test